index.html

```diff
@@ -0,0 +1,157 @@
+    </section>
+    <section class="first-page">
+        <header class="header-sect">
+            <a href="#"><i class="fas fa-bars fa-2xl" id="hamburger-menu"></i></a>
+        </header>
+        <article class="intro-sect">
+            <p class="heading-intro fw-bold">Stop Thinking Start Doing!</p>
+            <h2 class="fw-bold">Work-Sweat-Achieve</h2>
+            <h2 class="fw-bold"> Nothing Will Work Unless You Do</h2>
+            <h2 class="fw-bold">2023</h2>
+            <p class="intro-description">The Pain You Feel Today Will Be The Strength You Feel Tomorrow.</p>
+            <h3>2023.06.28(Wed)</h3>
+            <h4>@Kabul Afghanistan</h4>
+        </article>
+    </section>
+    <!-- Programs' Section -->
+    <section class="programs-sect" id="programs-section">
+        <h2 class="main-program">AHMADI GYM Program</h2>
+        <hr class="under-line">
+        <div class="program-activities">
+            <ul>
+                <li>
+                    <i class="fas fa-atom fa-2xl comments"></i>
+                    <h4>Trainings</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+                <li>
+                    <i class="fas fa-burn fa-2xl comments"></i>
+                    <h4>Moments</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+                <li>
+                    <i class="fas fa-brain fa-2xl comments"></i>
+                    <h4>Mindet</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+                <li>
+                    <i class="fas fa-balance-scale fa-2xl comments"></i>
+                    <h4>Team</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+                <li>
+                    <i class="fas fa-bahai fa-2xl comments"></i>
+                    <h4>Training Loop</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+            </ul>
+        </div>
+        <h4 class="whole-program"><a href="#" id="join-section">Join AHMADI GYM</a></h4>
+    </section>
+    
+    <!-- Featured Filmmakers Section -->
+    <section class="featured-sect" id="featured-section">
+    </section>
+
+<!-- Partners' Section -->
+  
+<!-- Footer Section -->
+    <section class="footer-sect">
+        <footer>
+            <h1 class="display-5 fw-bold p-3 m-3">AHMADI GYM</h1>
+            <div>
+            <p class="lead "> Your gym is meant to inspire a lifestyle in your members. Focus your efforts on running your gym, fitness center, <br> or studio.</p>
+            </div>
```

Comment:
- [ ] The `partner section` of your design is missing the proper template and it does not follow what we have in the original design. I would kindly suggest you work on it and add the necessary template and design.